tools/create_f2fs_image.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+SYZDIR=/usr/local/google/home/jiahengh/gopath/src/github.com/google/syzkaller
+lab1="-a 1 "
+lab2="-s 2 "
+lab3="-m "
```

Comment:
We should probably also test enabling some `-O` features to get more coverage in the kernel.